src/test/groovy/com/ofg/geb/AcceptanceSwaggerUISpec.groovy

```
@@ -0,0 +1,54 @@
+package com.ofg.geb
+
+import com.ofg.geb.pages.SwaggerUIHomePage
+import com.ofg.twitter.Application
+import geb.spock.GebSpec
+import org.springframework.boot.test.IntegrationTest
+import org.springframework.boot.test.SpringApplicationContextLoader
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.context.web.WebAppConfiguration
+
+@ContextConfiguration(loader = SpringApplicationContextLoader.class, classes = Application)
+@WebAppConfiguration
+@IntegrationTest("spring.profiles.active:dev,stubrunner.skip-local-repo:true")
+class AcceptanceSwaggerUISpec extends GebSpec {
+
```

Would it be useful to fail fast with `@StepWise`?